### ISSUE TITLE
autohide_sidebar.css reflows top buttons of sidebery when collapsed

### ISSUE THREAD

**Issue by guraltsev** [MrOtherGuy/firefox-csshacks#296]
Thank you for maintaining this repository. 

I added the autohide_sidebar.css but there seems to be something wrong with the current sidebery version (I use it for vertical tree-style tabs).

When the tab bar is collapsed when unhovered what happens is this:
 
![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/100e06c2-6893-42c3-878f-8c794d48dc86)

while it is like this when expanded:

![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/dff2dd40-67cf-486a-8b87-c64f2e9fb0e2)

I circled the weird space that appears. 

Interestingly, if I add new panels (control buttons) like this:
![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/c90c05c5-84da-4ebe-bc5e-5648df80651c)

When collapsed, it still gives:
![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/c482b1aa-f3b5-4e7e-8efb-0fcb56256145)

Actually, the more the "panels" the more the extra white space!

![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/9f8bc119-0a20-4611-bc5e-6c6953810ce7)
vs
![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/00628382-0c5d-4196-835e-f40d229ef464)

I think that the space for those icons gets reflowed. I tried looking through the code but I am too inexperienced to fix it.

Thank you!

**Comment by MrOtherGuy** [comment]
This sounds like something you need to change on Sidebery's side. It kind looks as if Sidebery uses a flex-container for that button-row and then wraps the items that can't fit horizontally - so essentially it wraps them all to separate lines. The solution would be to make it not do that.

**Comment by guraltsev** [comment]
Thanks for the head's up! I found this option that is relevant and actually solves the problem:

![image](https://github.com/MrOtherGuy/firefox-csshacks/assets/7040974/00b4025c-e3e6-4031-87eb-19a1fad2c295)

I will note this on the Sidebery's side so that people know of this issue. Actually, Sidebery can inject custom CSS. A possible solution to have both the icons on multiple lines and correct collapsing functionality would be to have the flex container have a fixed width.

Thank you again!

**Comment by MrOtherGuy** [comment]
Yeah, true - for so setups it would be necessary to have them on multiple lines. So yeah, it just has to not depend on the sidebar width, which is the "window width" for the sidebar document - or at least there needs to be a minimum width the flex-container can shrink to.

Still, that's something that Sidebery has to do. I can't do that from the autohide_sidebar side - at least not without breaking some other things.

**Comment by MrOtherGuy** [comment]
Based on what the @guraltsev described I'd say that on Sidebery options find "Layout > Show navigation bar in one line" and make sure that option is turned on.
